src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailControlModel.cs

```diff
@@ -385,11 +389,9 @@ private async Task<IReadOnlyList<PackageDependency>> GetDependencies(IProjectCon
         // Called after package install/uninstall.
         public abstract Task RefreshAsync(CancellationToken cancellationToken);
 
-        public event PropertyChangedEventHandler PropertyChanged;
-
         protected void OnPropertyChanged(string propertyName)
         {
-            PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
+            RaisePropertyChanged(propertyName);
```

Comment:
nit: I agree with not changing every instance in this PR (unless you feel motivated to do so), but changing just this one to be different is probably more confusing to the reader.

```suggestion
            OnPropertyChanged(propertyName);
```

Reply:
Is the suggestion to rename the RaisePropertyChanged function in ViewModelBase to OnPropertyChanged?

Reply:
Nope - the suggested change above is the only 1. It will then match all the other instances in this file.
Ideally, but a lot more changes required, just call RaisePropertyChanged directly in this file and get rid of OnPropertyChanged.

Reply:
I'll do that in a separate PR since there are 49 references to OnPropertyChanged in this file.